catalog-rest-service/src/main/resources/json/schema/metadataIngestion/databaseServiceProfilerPipeline.json

```diff
@@ -9,6 +9,27 @@
       "type": "string",
       "enum": ["Profiler"],
       "default": "Profiler"
+    },
+    "partitionDetails": {
```

Comment:
one question here. How would this config handle different requirements for different tables? I was thinking that this could maybe become part of the table.json itself 🤔

Reply:
@TeddyCr @pmbrull table.json already has tableParition https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-core/src/main/resources/json/schema/entity/data/table.json#L125

Reply:
@harshach thanks for pointing this out. If I am looking at the sink correctly, it seems that right now when we create table we are not passing this logic https://github.com/open-metadata/OpenMetadata/blob/ffcc88330cb08e6480147b38745119e5a0e34474/ingestion/src/metadata/ingestion/sink/metadata_rest.py#L196-L204

Reply:
@ayush-shah can you help here. I thought we added this for BigQuery, Hive etc..

Reply:
IMO these are two different elements:
1. What partition information do we extract as metadata from a Table, which as Teddy said I believe it is not being done.
2. Then at the profiler, what partitions we take into account for the sampling & profiling.

here my question was related to for example having a table partitioned by X, and we want to profile values A, B C; and then another table with the same X partition where we are only interested in profiling A, B.

Reply:
@harshach we had a logic in bigquery to fetch sample data for partition tables,
https://github.com/open-metadata/OpenMetadata/blob/0.10.1/ingestion/src/metadata/ingestion/source/bigquery.py#L131-L161, we actually had yet to implement details for partitions.

Reply:
@pmbrull you have a few scenarios:
1. you don't pass any `partitionField` value in the profiler workflow config and the profiler will pick up the first partition field when inspecting the table and will apply a time based partition -- works for multiple tables with different partition field but the same partition field type (i.e. date/datetime)
2. if you pass a `partitionField` then it will force the partition on this specific field and skip the inspection of the table to fetch metadata -- would work on multiple tables with the same partition field. If that partition field is not a date/datetime type then you'll need to pass the values to `partitionValues`. 
3. if you pass `partitionValues` then the partition will be made on those values (could be any types).

Based on these scenarios the profiling config needs to be adapted.

Reply:
@TeddyCr, I see. but then this configuration would be applied to all tables being profiled. Having that information at Table level might give more flexibility, similar to how profileSample works

Reply:
Yep, that makes total sense. We went that route as currently there is no support for partition details at the table level. Should we pause that one implement the support for table partition ingestion and finish up this one?

Reply:
@pmbrull thanks for hoping on a call. We'll move that partition details to the `test_suite` at the table level.